corehq/apps/app_manager/templates/app_manager/partials/add_item_templates.html

```diff
@@ -0,0 +1,126 @@
+{% load i18n %}
+
+<script type="text/html" id="js-popover-template-add-item">
+  <div class="popover popover-additem" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>
+</script>
+
+<script type="text/html" class="js-popover-template-add-item-content" data-slug="module">
+  <div>
+    <div class="pull-left">
+      <button type="button" class="popover-additem-option new-module" data-type="survey">
+        <i class="fa-regular fa-file"></i> {% trans "Surveys" %}
+        <p>{% blocktrans %}Collect data once.{% endblocktrans %}</p>
+      </button>
+    </div>
+    <div class="pull-left">
+      <button type="button" class="popover-additem-option new-module" data-type="case">
+        <i class="fa fa-bars"></i> {% trans "Case List" %}
+        <p>{% blocktrans %}Track items over time.{% endblocktrans %}</p>
+      </button>
+    </div>
+    {% if show_biometric %}
+      <div class="pull-left">
+        <button type="button" class="popover-additem-option new-module" data-type="biometrics">
+          <i class="fcc fcc-app-biometrics"></i> {% trans "Biometrics" %}
+          <p>{% blocktrans %}Identify clients by fingerprint.{% endblocktrans %}</p>
+        </button>
+      </div>
+    {% endif %}
+    {% if show_advanced %}
+      <div class="pull-left">
+        <button type="button" class="popover-additem-option new-module" data-type="advanced">
+          <i class="fa fa-flask"></i> {% trans "Advanced Menu" %}
+          <p>{% blocktrans %}Complex case management.{% endblocktrans %}</p>
+        </button>
+      </div>
+    {% endif %}
+    {% if show_report_modules %}
+      <div class="pull-left">
+        <button type="button" class="popover-additem-option new-module" data-type="report">
+          <i class="fa-regular fa-chart-bar"></i> {% trans "Report Menu" %}
+          <p>{% blocktrans %}Worker performance.{% endblocktrans %}</p>
+        </button>
+      </div>
+    {% endif %}
+    {% if show_shadow_modules %}
+      <div class="pull-left">
+        <button type="button" class="popover-additem-option new-module" data-type="shadow">
+          <i class="fa-regular fa-moon"></i> {% trans "Shadow Menu" %}
+          <p>{% blocktrans %}Mimic another menu.{% endblocktrans %}</p>
+        </button>
+      </div>
+    {% endif %}
+    {% if show_shadow_module_v1 %}
+      <div class="pull-left">
+        <button type="button" class="popover-additem-option new-module" data-type="shadow-v1">
```

Comment:
indentation

Reply:
basically anywhere on this page where there is a `data-` attribute, it should be on a new attribute line. thanks!